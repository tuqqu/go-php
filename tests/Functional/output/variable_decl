1 string 2 1 true
2 true 4
   1 2 3
1 true false 3
false
88
false
1
