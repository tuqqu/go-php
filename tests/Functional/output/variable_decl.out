test_var_decl
1 string 2 1 true
2 true 4
   1 2 3
1 true false 3
false
88
false
1
test_short_var_decl
0 true false 3
hello 65 true
false
88
false
0
test_mixed
0 true false 3
hello 65 true
